core/domain/draft_upgrade_services_test.py

```diff
@@ -183,7 +217,40 @@ def test_convert_states_v32_dict_to_v33_dict(self):
                         'value': False
                     }
                 }
-            }).to_dict())
+            }),
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'state2',
+                'property_name': 'widget_customization_args',
+                'new_value': {
+                    'choices': {
+                        'value': [
+                            '<p>1</p>',
+                            '<p>2</p>',
+                            '<p>3</p>',
+                            '<p>4</p>'
+                        ]
+                    },
+                    'maxAllowableSelectionCount': {
+                        'value': 1
+                    },
+                    'minAllowableSelectionCount': {
+                        'value': 1
+                    }
+                }
+            })]
+        # Migrate exploration to state schema version 33.
+        self.perform_and_assert_exploration_states_schema_migration('32', '33')
+        # Migrate the draft change list's state schema to the migrated
+        # exploration's schema.
+        migrated_draft_change_list_v33 = (
+            draft_upgrade_services.try_upgrading_draft_to_exp_version(
+                draft_change_list_v32, 1, 2, self.EXP_ID)
+        )
+        self.assertEqual(migrated_draft_change_list_v33[0].to_dict(),
+                         draft_change_list_v33[0].to_dict())
+        self.assertEqual(migrated_draft_change_list_v33[1].to_dict(),
+                         draft_change_list_v33[1].to_dict())
 
     def test_convert_states_v31_dict_to_v32_dict(self):
         draft_change_list = [
```

Comment:
Perhaps add _v31 suffix to the end, for consistency with everything else? And you'll probably need a note below to explain why it's unchanged.

Reply:
Added a note - is it necessary to add a note if the documentation is in draft_upgrade_services.py? I guess it's never bad to have the documentation in both places?

Reply:
Yes, I think what you have now is fine. Thanks!